### PR TITLE
check_prime added in JAVA

### PR DESCRIPTION
**Pull request by krishna-NIT**
check_prime.java added
returns true if no is prime otherwise returns false
Necessary Comments included,
Sample Input, Output & Complexity included 
for issue assigned to  me #157
by ShriKrushna Bhagwat
GSSoC 21' Contributor

### CODE REVIEW

**Review by krishna-NIT — COMMENTED** [review]
#157 Solved @nakul-19 @rudrakshi99 @plazzy99 take a look to code

**Comment by krishna-NIT** [comment]
@plazzy99 @rudrakshi99 Will you please help me with why my pull request showing actions needed,
please guide me

File: Code/Java/check_prime.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package primeno;
+import java.util.Scanner;
+/*
+Prime Numbers are those who dont's have any divisors other than 1 and itself,
```
Comment (plazzy99) [inline Code/Java/check_prime.java:4]: bring this comment outside and at the top of the code

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Code LGTM just make the requested changes

File: Code/Java/check_prime.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        Output: false
+
+        One Exception : 2 is the only even prime (Taken Special case)
+        Input: 2
+        Output: true
+
+        Time Complexity: O(n)
+        Space Complexity: O(1)
+
+ */
```
Comment (raksha009) [inline Code/Java/check_prime.java:57]: Plz add a new line here.

**Review by raksha009 — CHANGES_REQUESTED** [review]

File: Array/peakval/peakvalueinarray.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package peakval;
+import java.util.Scanner;
+/*
+Here we will Assume Peak element in Array is array[0] i.e 1st value,
```
Comment (plazzy99) [inline Array/peakval/peakvalueinarray.java:4]: take this comment at the top of the code

**Review by plazzy99 — CHANGES_REQUESTED** [review]
You need to raise 2 separate issues for resolving 2 different pr else you will lose points

**Comment by krishna-NIT** [comment]
@plazzy99 all correction in #720,
I am new to GitHub so, done this crazy stuffs earlier, sorry for that
tell me if any changes

**Comment by krishna-NIT** [comment]
@plazzy99 all correction in #720,
I am new to GitHub so, done this crazy stuffs earlier, sorry for that
my correct new &b accurate pull request at #720 
tell me if any changes

**Comment by krishna-NIT** [comment]
@raksha009 May I know, Why the tags are removed, I have done it in java & have made pull request

**Pull request closed without merging**